src/test/functional/tests/addUser.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { expect, test } from '@playwright/test';
+
+import { adminUserTestData } from '../data';
+import { LoginPage } from '../pages/LoginPage';
+import { PimPage } from '../pages/PimPage';
+
+let loginPage: LoginPage, pimPage: PimPage;
+
+test.describe('Adding a new employee', () => {
+  test.beforeEach(async ({ page }) => {
+    loginPage = new LoginPage(page);
+    pimPage = new PimPage(page);
+    await loginPage.initialize();
+    await loginPage.loginUser(adminUserTestData.userName, adminUserTestData.password);
+    await loginPage.navigateToSubPage('PIM');
+    await expect(page.getByRole('heading', { name: 'Employee Information' })).toBeVisible();
+  });
+
+  test.afterEach(async () => {
+    await loginPage.close();
+  });
+
+  test('Add new employee user with account  & fill new employee personal details', async () => {
+    await pimPage.addEmployeeWithLoginCredentials();
+    await pimPage.fillNewEmployeePersonalDetails();
+  });
```

Comment:
Where are the test assertions? So basically this test doesn't check anything other than opening personal details in flow and at the last step of the test displaying "Success". Don't you think it's a bit too little for user creation? I mean - please consider checking the user data that you provided... For example... that test would be green if a new field were added (and "nth" locators would be still in use) even if you in fact use "first name" value as a "hobby" :)